shuffle/cli/tutorials/HelloNFT.md

```diff
@@ -102,68 +102,197 @@ Let's try calling the `mint_nft` script function within the console.
 
 > let typeArguments: string[] = [];
 
-> let args: any[] = [contentUri];
+> let args: any[] = [move.Ascii(contentUri)];
```

Comment:
I would prefer mv.MoveType (or move.MoveType), but then I realized you can remove the type annotation completely

Reply:
What do you mean remove the type annotation completely?